### PR TITLE
[Live-Site] feat: join changes in events in live-site

### PR DESCRIPTION
**Pull request by satyam73**
This PR contains the changes that were needed for onboarding flow change in `live-site`.

### CODE REVIEW

**Review by Pratiyushkumar — COMMENTED** [review]
please give some context in the PR description as to what are you trying to solve and other stuff that can give the reviewer idea about your task

**Review by iamitprakash — CHANGES_REQUESTED** [review]

File: controllers/events.js [hunk 1]
```diff
@@ -92,14 +92,42 @@ const getAllEvents = async (req, res) => {
  * @throws {Error} If an error occurs while generating the token.
  */
 const joinEvent = async (req, res) => {
```
[the remark this answers is not in the document]
Comment (satyam73) [reply controllers/events.js:94]: Yes @iamitprakash , I've actually used the active event id and passed it to tokenService.getAuthToken() by destructuring and passed {roomId: eventId}, you can see line 101 I've assigned eventId with active event id

**Review by Pratiyushkumar — APPROVED** [review]
LGTM

**Review by yesyash — APPROVED** [review]

File: routes/events.js [hunk 2]
```diff
@@ -3,15 +3,48 @@ const router = express.Router();
...
 router.post("/join", eventsValidator.joinEvent, events.joinEvent);
 router.get("/:id", eventsValidator.getEventById, events.getEventById);
 router.patch("/", authenticate, eventsValidator.updateEvent, events.updateEvent);
-router.patch("/end", authenticate, eventsValidator.endActiveEvent, events.endActiveEvent);
-router.post("/:id/peers", authenticate, eventsValidator.addPeerToEvent, events.addPeerToEvent);
-router.patch("/:id/peers/kickout", authenticate, eventsValidator.kickoutPeer, events.kickoutPeer);
-router.post("/:id/codes", authenticate, eventsValidator.generateEventCode, events.generateEventCode);
-router.get("/:id/codes", authenticate, eventsValidator.getEventCodes, events.getEventCodes);
+router.patch(
+  "/end",
```
[the remark this answers is not in the document]
Comment (yesyash) [reply routes/events.js:22]: - Please make a ticket for this @satyam73

**Pull request merged**